en/docs/services/create-your-first-service.md

```diff
@@ -0,0 +1,181 @@
+# Create Your First Choreo Service
+
+To learn how to create a Choreo service, let's try a simple scenario. In this example, you are creating a service that consumes a public API that fetches the current active COVID-19 cases in a specified country, testing it, deploying it and then observing its performance.
+
+## Step 1: Create the Choreo service
+
+To create the Choreo service, follow the procedure below: 
+
+1. Access the Choreo Development Console via the following URL.
+
+    `https://console.choreo.dev/`
+    
+    Sign in using either your Google or Github credentials.
+    
+2. In the **Services** page, click **Create**.
+
+3. Under **Create with Choreo**, enter `covid-stats` as the name of your Choreo service.
+
+4. In the **Configure API Trigger** form, select **GET** as the HTTP method, and enter `activecases` as the relative path from host.
```

Comment:
Instead of bold, shall we use back tick for these words?

Reply:
Used the bold because the text is visible on the UI without the user entering it.

Reply:
Ack.